### PR TITLE
Dev 950

### PR DESCRIPTION
**Pull request by ahsinsiddique**

### CODE REVIEW

File: src/app/mail/mail-sidebar/mail-sidebar.component.ts [hunk 1]
```diff
@@ -33,12 +33,13 @@ export class MailSidebarComponent implements OnInit, OnDestroy {
 
   LIMIT = 3;
   EMAIL_LIMIT = 20;
-
+  PAGE = 1;
```
Comment (atifsaddique211f) [inline src/app/mail/mail-sidebar/mail-sidebar.component.ts]: This variable is unused, remove it, All of this page doesn't have any useful change, remove all changes from this page.

File: src/app/store/reducers/mail.reducers.ts [hunk 2]
```diff
@@ -215,6 +215,7 @@ export function reducer(
         noUnreadCountChange: true,
         canGetUnreadCount: true,
         decryptedSubjects: {},
+        pageNumber: 1,
```
Comment (atifsaddique211f) [inline src/app/store/reducers/mail.reducers.ts]: Remove page number as you don't need it anymore.

File: src/app/store/reducers/mail.reducers.ts [hunk 3]
```diff
@@ -251,7 +252,7 @@ export function reducer(
     }
 
     case MailActionTypes.SET_CURRENT_FOLDER: {
-      return { ...state, currentFolder: action.payload };
+      return { ...state, ...action.payload };
```
Comment (atifsaddique211f) [inline src/app/store/reducers/mail.reducers.ts]: change it back.

File: src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.ts [hunk 4]
```diff
@@ -121,7 +121,7 @@ export class GenericFolderComponent implements OnInit, AfterViewInit, OnDestroy
           }
           if (params.folder) {
             this.mailFolder = params.folder as MailFolderType;
-            this.store.dispatch(new SetCurrentFolder(this.mailFolder));
+            this.store.dispatch(new SetCurrentFolder({ currentFolder: this.mailFolder }));
```
Comment (atifsaddique211f) [inline src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.ts]: remove this change.

File: src/app/mail/mail-sidebar/mail-sidebar.component.html [hunk 5]
```diff
@@ -43,19 +43,19 @@
                         </ng-container>
                     </a>
                 </li>
-                <li class="mailbox-sidebar-nav-item"  routerLinkActive="active">
-                    <a routerLink="/mail/draft/page/1" class="mailbox-sidebar-nav-link" (click)="toggleMenu()">
+                <li class="mailbox-sidebar-nav-item"  [class.active]="mailState?.currentFolder === mailFolderType.DRAFT">
+                    <a routerLink="/mail/draft/page/1" class="mailbox-sidebar-nav-link" (click)="toggleMenu('draft')">
```
Comment (atifsaddique211f) [inline src/app/mail/mail-sidebar/mail-sidebar.component.html]: why did you pass `draft` in toggleMenu function

Comment (ahsinsiddique) [reply src/app/mail/mail-sidebar/mail-sidebar.component.html]: after changing issue resolving strategy, forgot to remove

**Pull request merged**